src/widgets/histogram/content-view.js

```diff
@@ -93,6 +93,10 @@ module.exports = cdb.core.View.extend({
       return;
     }
 
+    if (this._dataviewModel.changed.bins) {
+      this.binsCount = this._dataviewModel.get('bins');
```

Comment:
Good question, I don't know, I suppose there is a technical reason, @javierarce could you help me?